src/data/pokemon_graphics/front_pic_anims.h

```diff
@@ -13204,7 +13195,7 @@ const union AnimCmd *const *const gMonFrontAnimsPtrTable[NUM_SPECIES + 1] =
     [SPECIES_ALCREMIE_GIGANTAMAX]            = sAnims_AlcremieGigantamax,
     [SPECIES_COPPERAJAH_GIGANTAMAX]          = sAnims_CopperajahGigantamax,
     [SPECIES_DURALUDON_GIGANTAMAX]           = sAnims_DuraludonGigantamax,
-    [SPECIES_URSHIFU_GIGANTAMAX]             = sAnims_UrshifuGigantamax,
+    [SPECIES_URSHIFU_SINGLE_STRIKE_STYLE_GIGANTAMAX] = sAnims_UrshifuGigantamax,
```

Comment:
This should probably be `sAnims_UnshifuSingleStrikeStyleGigantamax` now?

Reply:
Done :)